src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_cloudresourcemanager.v1.json.baseline

```diff
@@ -0,0 +1,767 @@
+var cloudresourcemanager = google.cloudresourcemanager('v1');
+
+// Use Application Default Credentials for authentication when running locally.
+// For more information, see:
+// https://developers.google.com/identity/protocols/application-default-credentials
+google.auth.getApplicationDefault(function(err, authClient) {
+  if (err) {
+    console.log(err);
+    return;
+  }
+  if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+    var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+    authClient = authClient.createScoped(scopes);
+  }
+
+  var request = {
+
+    resource: {
+      // TODO: Add desired properties of the request body.
+    },
+
+    auth: authClient
+  };
+
+  var handlePage = function(err, response) {
+    if (err) {
+      console.log(err);
+      return;
+    }
+
+    var organizationsPage = response['organizations'];
+    if (!organizationsPage) {
+      return;
+    }
+    for (var i = 0; i < organizationsPage.length; i++) {
+      // TODO: Change code below to process each resource in `organizationsPage`:
+      console.log(JSON.stringify(organizationsPage[i], null, 2));
+    }
+
+    if (response.nextPageToken) {
+      request.pageToken = response.nextPageToken;
+      cloudresourcemanager.organizations.search(request, handlePage);
+    }
+  };
+
+  cloudresourcemanager.organizations.search(request, handlePage);
+});
+// BEFORE RUNNING:
+// ---------------
+// 1. If not already done, enable the Google Cloud Resource Manager API
+//    and check the quota for your project at
+//    https://console.developers.google.com/apis/api/cloudresourcemanager
+// 2. This sample uses Application Default Credentials for authentication.
+//    If not already done, install the gcloud CLI from
+//    https://cloud.google.com/sdk/ and run
+//    `gcloud beta auth application-default login`
+// 3. Install the Node.js client library and Application Default Credentials
+//    library by running `npm install googleapis --save`
+var google = require('googleapis');
+var cloudresourcemanager = google.cloudresourcemanager('v1');
+
+// Use Application Default Credentials for authentication when running locally.
+// For more information, see:
+// https://developers.google.com/identity/protocols/application-default-credentials
+google.auth.getApplicationDefault(function(err, authClient) {
+  if (err) {
+    console.log(err);
+    return;
+  }
+  if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+    var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+    authClient = authClient.createScoped(scopes);
+  }
+
+  var request = {
+
+    // TODO: Change placeholder below to desired parameter value for the `setIamPolicy` method:
+
+    // REQUIRED: The resource for which the policy is being specified. `resource` is usually specified as a
+    // path. For example, a Project resource is specified as `projects/{project}`.
+    resource_: 'organizations/{MY-ORGANIZATION}',
```

Comment:
All request bodies in Node.js are called `resource`, but Cloud Resource Manager also has a field called `resource`. The library owners decided to resolve the conflict by renaming the field to `resource`.

This override is applied in NodeJSDiscoveryContext, so once the Node.js MVVM flow gets merged, it will be applied from the overrides file.